pb_plugins/protoc_gen_dcsdk/docs.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+
+class Docs(object):
```

Comment:
I think you don't need the `object` if it's only used with Python 3.

Reply:
Oh, didn't know that, I was blindly copying the other files :sweat_smile:. I'll change that everywhere then.